brclient: panic: runtime error when relaying news feed

Getting the following when relaying a news feed post. The app crashes, but upon relaunching the post appears as successfully relayed.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x1121943]

goroutine 1900 [running]:
[github.com/companyzero/bisonrelay/client.(*RemoteUser).Nick(...)](http://github.com/companyzero/bisonrelay/client.(*RemoteUser).Nick(...))
	[github.com/companyzero/bisonrelay/client/remoteuser.go:142](http://github.com/companyzero/bisonrelay/client/remoteuser.go:142)
main.newAppState.func17(0x0?, {{0xfc, 0xfe, 0x4f, 0x51, 0x21, 0xd7, 0x36, 0x59, 0x5f, ...}, ...}, ...)
	[github.com/companyzero/bisonrelay/brclient/appstate.go:2366](http://github.com/companyzero/bisonrelay/brclient/appstate.go:2366) +0x43
[github.com/companyzero/bisonrelay/client.(*Client).relayPost(0xc000bf6c40,](http://github.com/companyzero/bisonrelay/client.(*Client).relayPost(0xc000bf6c40,) {0x1a, 0x83, 0x70, 0x15, 0x73, 0xd5, 0x6d, 0x73, 0x92, ...}, ...)
	[github.com/companyzero/bisonrelay/client/client_posts.go:908](http://github.com/companyzero/bisonrelay/client/client_posts.go:908) +0x447
[github.com/companyzero/bisonrelay/client.(*Client).RelayPostToSubscribers(0xc000bf6c40,](http://github.com/companyzero/bisonrelay/client.(*Client).RelayPostToSubscribers(0xc000bf6c40,) {0x1a, 0x83, 0x70, 0x15, 0x73, 0xd5, 0x6d, 0x73, 0x92, ...}, ...)
	[github.com/companyzero/bisonrelay/client/client_posts.go:965](http://github.com/companyzero/bisonrelay/client/client_posts.go:965) +0xed
main.(*appState).relayPostToAll(0xc0003a2000, {0x1a, 0x83, 0x70, 0x15, 0x73, 0xd5, 0x6d, 0x73, 0x92, ...}, ...)
	[github.com/companyzero/bisonrelay/brclient/appstate.go:1677](http://github.com/companyzero/bisonrelay/brclient/appstate.go:1677) +0x145
created by main.postWindow.Update
	[github.com/companyzero/bisonrelay/brclient/postwin.go:792](http://github.com/companyzero/bisonrelay/brclient/postwin.go:792) +0x1cbd

Ubuntu 22.04, brclient-linux-amd64-0.1.1
```

--- a/brclient/appstate.go
+++ b/brclient/appstate.go
@@ -2357,13 +2357,15 @@ func newAppState(sendMsg func(tea.Msg), lndLogLines *sloglinesbuffer.Buffer,
 
 			// user == nil when event is a post being relayed for
 			// the first time.
+			nick := "(no nick)"
 			if user != nil {
 				if user.IsIgnored() {
 					return
 				}
+				nick = user.Nick()
 			}
 			as.diagMsg("Received post %q (%s) from %q",
-				summ.Title, summ.ID, user.Nick())
+				summ.Title, summ.ID, nick)
 
 			// Store new post.
 			as.loadPosts()

